llvm/lib/CodeGen/TwoAddressInstructionPass.cpp

```diff
@@ -80,6 +80,13 @@ EnableRescheduling("twoaddr-reschedule",
                    cl::desc("Coalesce copies by rescheduling (default=true)"),
                    cl::init(true), cl::Hidden);
 
+// Limit the number of rescheduling visits to dependent instructions.
+// FIXME: Arbitrary limit to reduce compile time cost.
+static cl::opt<unsigned>
+    MaxVisits("twoaddr-visit-limit", cl::Hidden, cl::init(10),
```

Comment:
Should we include "reschedule" in the option name? "twoaddr-reschedule-visit-limit"?

Reply:
I feel like I am not the person who should make that decision

Reply:
`twoaddr-reschedule-visit-limit` sgtm